packages/soci-snapshotter/soci-config-toml

```diff
@@ -0,0 +1,60 @@
+[required-extensions]
```

Comment:
For the lack of a better place to put this, we should fix the SOCI snapshotter service and make it use the HTTP proxy env var, right?

See: https://github.com/bottlerocket-os/bottlerocket-core-kit/pull/564